gwsurrogate/surrogate.py

```diff
@@ -966,21 +1085,38 @@ def __call__(self, q, M=None, dist=None, theta=None,phi=None,
        For circular orbits, the binary's orbital angular momentum is taken to
        be the z-axis. Theta and phi is location on the sphere relative to this
        coordinate system. """
-
-
+   
     if (not self.use_orbital_plane_symmetry) and fake_neg_modes:
       raise ValueError("if use_orbital_plane_symmetry is not assumed, it is not possible to fake m<0 modes")
-
+        
     ### deduce single mode dictionary keys from ell, m and fake_neg_modes input ###
     modes_to_evaluate = self.generate_mode_eval_list(ell,m,fake_neg_modes)
 
     if mode_sum and (theta is None and phi is None) and len(modes_to_evaluate)!=1:
       raise ValueError('Trying to sum modes without theta and phi is a strange idea')
 
+    # For models where the higher modes have been modeled in the coorbital frame, 
+    # it is necessary to have the (2,2) mode as the first index of modes_to_evaluate
+    # otherwise, we do not have the information regarding the orbital phase
+    # crucial to transform the higher modes from coorbital frame to inertial frame
+    # this line of code guarantees that 
+    #   (1) (2,2) is always included in the modes to evaluate
+    #   (2) (2,2) is the first mode in modes_to_evaluate
+    if self.surrogateID=='BHPTNRSur1dq1e4':
```

Comment:
Something to consider:
1. This is a bit wasteful because you are also evaluating Amp22 which is not necessary to go from coorb to intertial. Instead, is it easy to make the code always evaluate phi22, because you always use it no matter what the mode content is?
2. This one is more serious. If I ask for the hp/hc at some incl/phi_ref, with only the (2,1) mode. Will your code return it with (2,1) and (2,2)? That's not good. If you are forced to do this, the way to do it is to raise an error if (2,2) is missing. Then, the user it at least aware of this.

Reply:
Okay - I think both the ones are relatively easier to implement. I have to think a bit how to do that. It always means loading 22 mode (as well as any other mode user might want) because that's how the SingleModeSurrogateIO works. 
If we force the user to include 22 mode, we always have the orbital phase info